src/Process/Resource/CopyImageProcess.php

```diff
@@ -135,7 +135,12 @@ protected function downloadImage(DomNode $node)
         if (!$this->fsio->isDir($dir)) {
             $this->fsio->mkdir($dir);
         }
-        $this->fsio->put($file, $this->fsio->get($originFile));
+
+        if (file_exists($originFile)) {
+            $this->fsio->put($file, $this->fsio->get($originFile));
+        } else {
+            $this->logger->warning("      Images {$originFile} does not exists.");
```

Comment:
Please replace `Images` with `Image` and `exists` with `exist`. It's only one file.